tasks/init.rb

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative '../../ruby_task_helper/files/task_helper'
+
+require 'net/http'
+require 'openssl'
```

Comment:
If we expect this to be used in plans and not directly - or really even if we expect it to be used directly - I think it makes sense to wrap this in a plan that ensures these gems are available.